tests/sentry/utils/email/test_backend.py

```diff
@@ -0,0 +1,17 @@
+from sentry.testutils import TestCase
+from sentry.utils.email.backend import get_mail_backend
+
+
+class MiscTestCase(TestCase):
```

Comment:
Why have this class and the class in `tests/sentry/utils/email/test_address.py` be named the same? I can't tell if this was a mistake or intentional.

Reply:
We were originally testing both functions in the same class. I accidentally kept the old name for both instead of coming up with more descriptive names.